### PR TITLE
E_CLASSROOM-263 [FE] [Admin] [improvement] Categories description width and logout functionality

### PR DESCRIPTION
**Pull request by ErickDoroy**
### Issue Link
- https://framgiaph.backlog.com/view/E_CLASSROOM-263
### Definition of Done
- [x] Make the description column width to 300px
- [x] Implement logout functionality
- [x] Redirect to login page

#### Main note
- http://localhost:3003/admin/login
- http://localhost:3003/admin/categories
#### Pages Affected
- 

### Scenarios/ Test Cases
- [x] Make the description column width to 300px
- [x] Implement logout functionality
- [x] Redirect to login page
#### User Interface
- N/A
#### User Experience 
- N/A
#### Functionality
- N/A
### Screenshots
admincategory

### CODE REVIEW

File: src/components/NavigationSideBar/index.js [hunk 1]
```diff
@@ -1,14 +1,32 @@
...
 
 const NavigationSideBar = () => {
+  const toast = useToast();
+
+  const onLogout = async () => {
+    toast('Processing', 'Logging out...');
+
+    try {
+      await AuthApi.logout();
+      toast('Success', 'Successfully Logged Out.');
```
Comment (karlolee50) [inline src/components/NavigationSideBar/index.js]: no need for successful toaster during login/logout

File: src/components/NavigationSideBar/index.js [hunk 2]
```diff
@@ -1,14 +1,32 @@
...
+      Cookies.remove('access_token');
+      window.location = '/admin/login';
+    } catch (error) {
+      console.log(error.response);
```
Comment (karlolee50) [inline src/components/NavigationSideBar/index.js]: use toaster for error

**Review by karlolee50 — CHANGES_REQUESTED** [review]
Review time: 10 minutes
Comment/s: 2

**Comment by catampongan** [comment]
## Approved PR #99 

- [x] Able to minimize the width of description column
- [x] Implemented logout functionality
----------
Screenshot 2022-02-09 192014
logout admin

------------
- Total Test Time: 0.25 hr
- Total Iterations: 0
- Total Comments: 0

**Review by karlolee50 — APPROVED** [review]

**Pull request merged**